src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -5,19 +5,19 @@
 public enum KeyBindingCategory {
 
     FILE(
-            Localization.menuTitle("File")),
+            Localization.lang("File")),
```

Comment:
I would not require the translators to replace terms twice. Why not just removing the `&` after getting the translation from `menuTitle`?

Reason: You are displaying menu strings. And menu strings are contained in `Menu_X.properties`. It might happen that a menu string is translated differently in JAbRef_*.properties. Using `Menu_*.properties` avoids that issue.